public/js/scripts.js

```diff
@@ -1,56 +1,93 @@
-function bookSearch(){
-
+function bookSearch () {
     //user input stored in var
     const find = document.getElementById('textName').value;
     document.getElementById('results').innerHTML = "";
+    asyncReq(find);
+}
+
+function asyncReq (searchQuery) {
+  $.ajax({
+    //properties - query + user input, data type, success, request type
+    url: 'https://www.googleapis.com/books/v1/volumes?q=' + searchQuery,
+    dataType: 'json',
+    success: function (data) {
+      successResult(data);
+    },
+    type: 'GET'
+  })
+}
+
+function successResult (data) {
+
+  /* .map method returns a new array,  markUp loops over each book and creates a new array item, the first param must be a function and its param is the array item `book` that will be looped over. Then, .join converts an array into a string (empty string passed through .join allows empty space to be printed).Then call 'printTitle' function to print string. Finally add .innerHTML so that it adds the mark up to the page once.
+  Template literals `${string}` are used to allow embedded expressions.
+   */
+
+
+    const markUp = data.items.map(function(book){
+      return ` 
+        <div>
+          ${printTitle(book.volumeInfo.title)}
+          ${printAuthor(book.volumeInfo.authors)}
+          ${displayCover(book.volumeInfo.imageLinks.smallThumbnail)}
+          ${printSynopsis(book.volumeInfo.description)}
+          ${printPublisher(book.volumeInfo.publisher)}
+          ${linkPreview(book.volumeInfo.previewLink)}
+        </div>
+      `
+    }).join('');
+
+    results.innerHTML = markUp;
+}
+
+function printTitle(title) {
```

Comment:
`print` is a word that is often used to say "write out to console". Maybe something descriptive like `getTitleMarkup()` could help disambiguate things.